app/tests/test_search.py

```diff
@@ -18,6 +18,7 @@ def verify_search_transferring_body_header_row(data):
     expected_row = (
         [
             "Series",
+            "Series / File name / Consignment ref",
```

Comment:
so we expect to see words series, file name and consignment twice?

Reply:
We're using CSS to only display one or the other for Desktop / Mobile. The HTML does contain both headers though to account for this.

Reply:
so does it look funky if user has css disabled?

Reply:
this does not look correct, should not be two sets for column headers , also the column header should be 'Consignment reference' when ever used

Reply:
@pgandhizaizi Updated to test for desktop column headers

Reply:
cleared the doubts after a call, approving the PR